test/golden/library_example.golden

```diff
@@ -313,6 +314,7 @@
 /ruby/google-cloud-ruby/google-cloud-library_example/lib/google/cloud/example/library/credentials.rb
 /ruby/google-cloud-ruby/google-cloud-library_example/lib/google/cloud/example/library/v1.rb
 /ruby/google-cloud-ruby/google-cloud-library_example/lib/google/cloud/example/library/v1/doc/google/example/library/v1/library.rb
+/ruby/google-cloud-ruby/google-cloud-library_example/lib/google/cloud/example/library/v1/doc/overview.rb
```

Comment:
@geigerj Jacob, please confirm this change is expected.

Reply:
Yes, this is expected. (It is a newly introduced file.)